lib/services/applicationinsightsClient/README.md

```diff
@@ -0,0 +1,49 @@
+---
+uid: azure-applicationinsights
+summary: *content
+
+---
+# Microsoft Azure SDK for Node.js - ApplicationInsightsDataClient
+This project provides a Node.js package for accessing Azure. Right now it supports:
+- **Node.js version 6.x.x or higher**
+
+## Features
+
+
+## How to Install
+
+```bash
+npm install azure-applicationinsights
+```
+
+## How to use
+
+### Authentication, client creation and get metrics as an example.
+
+```javascript
+const msRest = require("ms-rest");
+const ApplicationInsightsDataClient = require("azure-applicationinsights");
+const token = "<access_token>";
```

Comment:
From where would one get the access token. Please provide that info. Aren't customers suppose to use the standard msRestAzure.loginWithServicePrincipal(..) method for authentication?